scheds/rust/scx_lavd/src/bpf/main.bpf.c

```diff
@@ -2321,69 +2677,150 @@ s32 BPF_STRUCT_OPS(lavd_init_task, struct task_struct *p,
 	 * tasks.
 	 */
 	if (args->fork) {
-		struct sys_cpu_util *cutil_cur;
-		static u64 load_ideal;
-
-		load_ideal = get_task_load_ideal(p);
-		cutil_cur = get_sys_cpu_util_cur();
+		struct sys_stat *stat_cur = get_sys_stat_cur();
+		u64 load_ideal = get_task_load_ideal(p);
 
-		__sync_fetch_and_add(&cutil_cur->load_ideal, load_ideal);
+		__sync_fetch_and_add(&stat_cur->load_ideal, load_ideal);
 	}
 
 	return 0;
 }
 
-s32 BPF_STRUCT_OPS_SLEEPABLE(lavd_init)
+static int calloc_cpumask(struct bpf_cpumask **p_cpumask)
 {
-	struct bpf_timer *timer;
-	u64 now;
-	u32 key = 0;
-	int cpu;
-	int err;
+	struct bpf_cpumask *cpumask;
+	cpumask = bpf_cpumask_create();
+	if (!cpumask)
+		return -ENOMEM;
+	bpf_cpumask_clear(cpumask);
```

Comment:
Just FYI: This isn't technically necessary. `bpf_cpumask_create()` will memset the cpumask to 0 before returning it.

Reply:
Fixed. Thanks!